### ISSUE TITLE
Strange bug with wires and logic - game or mod dont see a part of scheme(1.16.5)

### ISSUE THREAD

**Issue by anrej0705** [MrTJP/ProjectRed#1655]
Hi coders)) I found a strange bug occuring in big schemes with high element density. It feels like the signal just gets stuck in the wires or game dont see a part of scheme/logic gates. With all this, there is a possibility to debug scheme without quiting world - i can simply break and put some elements of my scheme and the bug will disappear

How to reproduce and debug(only in my case and in my world) you can see in video:
https://www.youtube.com/watch?v=RtBcB8isTWY

minecraft map from video:
https://drive.google.com/file/d/1um_RT75IEVY7lCJzDC8slhDj0RtwKWfK/view?usp=sharing

Mod list:
1. Forge 36.1.31
2. WorldEdit 7.2.5+57d5C9
3. Architectury: 1.19.27
4. FTB-Library: 1605.3.1-build.48
5. JEI: 7.7.1.110
6. Reap: 1.16.5-1.0.1
7. CCL: 4.0.2.427
8. CBMultipart: 3.0.1.119
9. CBMicroblock: 3.0.1.119
10. ProjectRed-Core: 4.1.0-beta-38
11. ProjectRed-Transmission: 4.1.0-beta-38
12. ProjectRed-Exploration: 4.1.0-beta-38
13. ProjectRed-Integration: 4.1.0-beta-38
14. ProjectRed-Illumintaion: 4.1.0-beta-38
15. IronChests: 1.16.4-11.2.1.0

**Comment by MrTJP** [comment]
Very impressive build :)

Did you happen to use worldedit to place the gates that are getting stuck?

**Comment by anrej0705** [comment]
Yep, i use worldedit to place "ram cell" and check the correctness of work after each copy. Everything else except this "field" I built manually

**Comment by anrej0705** [comment]
gates that are getting stuck i place manually, "stucking" gates and wires were seen only in display building, but display I was build fully manually

**Comment by MrTJP** [comment]
When you uploaded that map, was it saved before or after you fixed the issue by breaking the gates?

**Comment by anrej0705** [comment]
i was upload map immediately after writing video.In last of part video(last minute on video) i not fix gates, left everything as it is

this bug is observed every time I join the world

**Comment by anrej0705** [comment]
you can check it by joing to my map with the list of mods that I had

**Comment by anrej0705** [comment]
I found an easier way to fix gates in my map
i just replace all repeaters and single-pulse generator, which was show on video at time 4:58
after this actions, my scheme was begin to work correctly

**Comment by MrTJP** [comment]
Thanks, that is helpful. I'll be able to investigate more in a few days.

**Comment by anrej0705** [comment]
Do you already have any progress in investigating this bug?
